### ISSUE TITLE
Question about pretrained model

### ISSUE THREAD

**Issue by Noba1anc3** [Viredery/tf-eager-fasterrcnn#8]
Hello,
I am wondering if the pretrained model of Faster RCNN is trained on ImageNet?
Thank you.

**Comment by Viredery** [comment]
Hi @Noba1anc3 
You can check [this](https://github.com/matterport/Mask_RCNN). I used this pre-trained weights and just deleted the mask branch.
